src/Agent.php

```diff
@@ -40,7 +40,11 @@ public function __construct(array $config, float $request_start_time)
     public function registerInitCollectors(): void
     {
         // Laravel init collector
-        $this->addCollector(app(FrameworkCollector::class));
+        if ('cli' !== php_sapi_name()) {
```

Comment:
Thought: if the collector is not initialised for cli requests should it still be called a "FrameworkCollector"?

Reply:
it is still a FrameworkCollector, but since the framework only loads once for workers, not sure about the value that it adds. I would like to measure artisan commands as well, maybe we can revisit the approach at that point.